catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/CachingFederationStrategy.java

```diff
@@ -264,6 +269,7 @@ private QueryResponse sourceFederate(List<Source> sources, final QueryRequest qu
                     if (source instanceof CatalogProvider && SystemInfo.getSiteName()
                             .equals(source.getId())) {
                         finalQueryRequest = getRequestWithTagsFilter(modifiedQueryRequest);
+                        finalQueryRequest = getRequestWithValidityFilter(finalQueryRequest);
```

Comment:
Should we chain the calls instead of assigning to `finalQueryRequest` twice in a row?

```
finalQueryRequest = getRequestWithValidityFilter(getRequestWithTagsFilter(modifiedQueryRequest));
```